lib/rules/fluent-chaining.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * @fileoverview Enforce spacing for chained calls
+ */
+
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+   create: function(context) {
+      var sourceCode = context.getSourceCode(),
+          lines = sourceCode.getLines();
+
+      //--------------------------------------------------------------------------
+      // Helpers
+      //--------------------------------------------------------------------------
+
+      function lineIndent(lineNumber) {
+         var line = lines[lineNumber - 1];
+
+         if (line === undefined) {
+            throw new Error('Invalid lineNumber ' + lineNumber + ' supplied to lineIndent');
+         }
+
+         return line[0] === ' ' ? line.match(/[^ ]/).index : 0;
```

Comment:
This means that the rule can only be used by folks who are using spaces for their indentation and not tabs, right? Would it be easy to support spaces or tabs? If so, we should do that.